xlml/utils/gpu.py

```diff
@@ -276,6 +299,7 @@ def wait_for_resource_creation(operation_name: airflow.XComArg):
                 "Error during resource creation: [Code:"
                 f" {operation.http_error_status_code}]:"
                 f" {operation.http_error_message}"
+                f" {operation.error}"
```

Comment:
Do you have an example for this `operation.error`?

Reply:
The error looks like this:  https://screenshot.googleplex.com/3cYYyU85shBeF3d 
I feel it might give more info though.

Reply:
Thank you!